docfx/docfx.json

```diff
@@ -208,6 +226,15 @@
       ],
       "dest": "metadata/Pulumi.Mailgun"
     },
+    {
+      "src": [
+        {
+          "src": "../../pulumi-mongodbatlas/sdk/dotnet",
+          "files": ["Pulumi.*.csproj"]
+        }
+      ],
+      "dest": "metadata/Pulumi.Mongodbatlas"

```

Comment:
I got this name by looking at our .NET SDK for this provider and checked the namespace for one of the C# classes.